src/Tooltip.js

```diff
@@ -25,7 +25,8 @@ const Tooltip = (props) => {
                 {React.cloneElement(newLabel, {
                     htmlFor: id,
                     className: classNames('mdl-tooltip', {
-                        'mdl-tooltip--large': large
+                        'mdl-tooltip--large': large,
+                        [`mdl-tooltip--${position}`]: true
```

Comment:
should use `typeof position !== 'undefined'`, because we don't want a useless css class called `mdl-tooltip--undefined` when the position is not specified ;-)